examples/weather_station/src/chip_project_config.h

```diff
@@ -16,18 +16,32 @@
 #pragma once
 
 /* Use a default pairing code if one hasn't been provisioned in flash. */
-#define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_PIN_CODE 20202021
-#define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xF00
+#define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_PIN_CODE 28073715
+#define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR 0xE46
 
 #define CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL 1000_ms32
 
 /* Configure device configuration with exemplary data */
 #define CHIP_DEVICE_CONFIG_DEVICE_VENDOR_NAME "Nordic"
 #define CHIP_DEVICE_CONFIG_DEVICE_PRODUCT_NAME "WeatherStation"
-#define CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_PRODUCT_REVISION_STRING                                  \
-	"Prerelease weather station device"
+#define CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_PRODUCT_REVISION_STRING "Prerelease weather station device"
```

Comment:
I think these are now taken from Kconfig, right? It's worth setting it to some "Matter SVE" or something like that.